com.github.utsushi.Utsushi.json

```diff
@@ -38,63 +42,117 @@
                     "type": "archive",
-                    "url": "http://ftp.gnome.org/pub/GNOME/sources/cairomm/1.12/cairomm-1.12.0.tar.xz",
-                    "sha256": "a54ada8394a86182525c0762e6f50db6b9212a2109280d13ec6a0b29bfd1afe6"
+                    "url": "https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.8.tar.xz"
                 }
+            ],
+            "config-opts": [
+                "--disable-documentation"
+            ],
+            "cleanup": [
+                "/lib/sigc++-2.0"
             ]
         },
         {
             "name": "glibmm",
             "sources": [
                 {
+                    "sha256": "afb96202491485d3f44102d98219a172da2d3f9eb78fce3cfb92559ba496e499",
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.3.tar.xz"
+                }
+            ],
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuild-documentation=false",
+                "-Dbuild-examples=false"
+            ],
+            "cleanup": [
+                "/lib/glibmm-2.4",
+                "/lib/giomm-2.4",
+                "/lib/libglibmm_generate_extra_defs*"
+            ]
+        },
+        {
+            "name": "cairomm",
+            "sources": [
+                {
+                    "sha256": "7dab16c5698acb284f178048337465b463fbed01c49e10bfe48acb0a852d873e",
                     "type": "archive",
-                    "url": "http://ftp.gnome.org/pub/GNOME/sources/glibmm/2.64/glibmm-2.64.2.tar.xz",
-                    "sha256": "a75282e58d556d9b2bb44262b6f5fb76c824ac46a25a06f527108bec86b8d4ec"
+                    "url": "https://gitlab.freedesktop.org/cairo/cairomm/-/archive/1.14.3/cairomm-1.14.3.tar.gz"
                 }
+            ],
+            "config-opts": [
+                "--disable-documentation"
+            ],
+            "cleanup": [
+                "/lib/cairomm-1.0"
             ]
         },
         {
             "name": "pangomm",
             "sources": [
                 {
+                    "sha256": "57442ab4dc043877bfe3839915731ab2d693fc6634a71614422fb530c9eaa6f4",
                     "type": "archive",
-                    "url": "http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.42/pangomm-2.42.1.tar.xz",
-                    "sha256": "14bf04939930870d5cfa96860ed953ad2ce07c3fd8713add4a1bfe585589f40f"
+                    "url": "https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.2.tar.xz"
                 }
+            ],
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuild-documentation=false"
+            ],
+            "cleanup": [
+                "/lib/pangomm-1.4"
             ]
         },
         {
             "name": "atkmm",
             "sources": [
                 {
+                    "sha256": "a0bb49765ceccc293ab2c6735ba100431807d384ffa14c2ebd30e07993fd2fa4",
```

Comment:
why change the order?

Reply:
The two order changes were unintentional, I'll fix it. Thanks.